src/loaders/raster-json-loaders/RasterJsonAsImageLoader.js

```diff
@@ -0,0 +1,44 @@
+import { AbstractLoaderError } from '../errors';
```

Comment:
It's frustrating that this and `RasterJsonAsImageLoader` are almost identical...I've definitely forgotten to update one or the other of bitmask/normal images before....I tried writing something to abstract this but I'm not sure it's worth it.  It seems like you'd need multiple methods etc. and the logic you're refactoring out would be minimal....